src/main/java/org/kuali/kfs/module/purap/document/service/impl/PaymentRequestServiceImpl.java

```diff
@@ -0,0 +1,2108 @@
+import org.kuali.kfs.core.api.config.property.ConfigurationService;
+import org.kuali.kfs.core.api.datetime.DateTimeService;
+import org.kuali.kfs.core.api.util.type.KualiDecimal;
+import org.kuali.kfs.kew.api.exception.WorkflowException;
+import org.kuali.kfs.kim.api.identity.Person;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * This class provides services of use to a payment request document
+ */
+/**
+ * CU customization: backport FINP-8270. This was backported to the 1/28/2021
+ * version of this file. This change can be removed with the 3/2/2022 upgrade.
+ */
+public class PaymentRequestServiceImpl implements PaymentRequestService {
+
+	private static final Logger LOG = LogManager.getLogger();
+
+	protected AccountsPayableService accountsPayableService;
+	protected BankService bankService;
+	protected BusinessObjectService businessObjectService;
+	protected ConfigurationService configurationService;
+	protected DataDictionaryService dataDictionaryService;
+	protected DateTimeService dateTimeService;
+	protected DisbursementVoucherValidationService disbursementVoucherValidationService;
+	protected DocumentService documentService;
+	protected FinancialSystemDocumentService financialSystemDocumentService;
+	protected KualiRuleService kualiRuleService;
+	protected NegativePaymentRequestApprovalLimitService negativePaymentRequestApprovalLimitService;
+	protected NoteService noteService;
+	protected ParameterService parameterService;
+	protected PaymentRequestDao paymentRequestDao;
+	protected PurapAccountingService purapAccountingService;
+	protected PurapService purapService;
+	protected PurApWorkflowIntegrationService purapWorkflowIntegrationService;
+	protected PurchaseOrderService purchaseOrderService;
+	protected UniversityDateService universityDateService;
+	protected VendorService vendorService;
+
+	@Override
+	@Deprecated
+	public Collection<PaymentRequestDocument> getPaymentRequestsToExtractByCM(String campusCode,
+			VendorCreditMemoDocument cmd) {
+		LOG.debug("getPaymentRequestsByCM() started");
+		Date currentSqlDateMidnight = dateTimeService.getCurrentSqlDateMidnight();
+		List<PaymentRequestDocument> paymentRequestIterator = paymentRequestDao.getPaymentRequestsToExtract(campusCode,
+				null, null, cmd.getVendorHeaderGeneratedIdentifier(), cmd.getVendorDetailAssignedIdentifier(),
+				currentSqlDateMidnight);
+
+		return filterPaymentRequestByAppDocStatus(paymentRequestIterator, PaymentRequestStatuses.APPDOC_AUTO_APPROVED,
+				PaymentRequestStatuses.APPDOC_DEPARTMENT_APPROVED);
```

Comment:
This overlay contains several formatting changes that do not line up with the 01/28/2021 version of the file. For example, it looks like this "return" statement is meant to span three lines instead of two. Can you please verify that the correct file version was overlayed?

Reply:
Chad, thank you for noticing. I updated the file.